smartcontractkit/external-adapters-js
# PoR Address List EA

## Closes 34889

## Description

This PR implements the new PoR Address list EA that fetches a list of custodial addresses that support a PoR feed for a wrapped token

## Changes

- Implement `por-address-list` EA

## Steps to Test

1. Set required `RPC_URL` env var.  This EA was tested using contracts deployed on Rinkeby
2. Send request below
3. Verify successful response
4. View the contract on **Rinkeby** Etherscan and verify addresses match what the EA responds with

Request
```
{
    "id": "1",
    "data": {
        "contractAddress": "0x203E97cF02dB2aE52c598b2e5e6c6A778EB1987B"
    }
}
```

## Quality Assurance
